webservices/common/models/itemized.py

```diff
@@ -47,9 +45,22 @@ class BaseRawItemized(db.Model):
     back_reference_schedule_name = db.Column('br_sname', db.String)
 
     @hybrid_property
+    def report_type(self):
+        return self.filing.form_type
+
+    @hybrid_property
+    def cycle(self):
+        return self.load_timestamp.year
+
+    @hybrid_property
     def memoed_subtotal(self):
         return self.memo_code == 'X'
 
+    @hybrid_property
+    def fec_election_type_desc(self):
+        election_map = {'P': 'PRIMARY', 'G': 'GENERAL', 'O': 'OTHER'}
+        return election_map.get(str(self.pgo).upper()[0])
```

Comment:
Is `self.pgo` (in the models that utilize this base class) guaranteed to always have a value that you can pull the first character out of?